<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsmInfoSettingsPageBase</class>
 <widget class="QWidget" name="OsmInfoSettingsPageBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="overpass_group">
     <property name="title">
      <string>Overpass server</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="endpoint_laout">
        <item>
         <widget class="QComboBox" name="endpoint_combobox"/>
        </item>
        <item>
         <widget class="QToolButton" name="endpoint_button">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="query_group">
     <property name="title">
      <string>Query settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="nearby_checkbox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Fetch nearby features</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enclosing_checkbox">
        <property name="text">
         <string>Fetch enclosing features</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="timeout_label">
          <property name="text">
           <string>Timeout</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="timeout_spinbox"/>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="distance_spinbox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="distance_label">
          <property name="text">
           <string>Distance</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="other_group">
     <property name="title">
      <string>Other</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="debug_checkbox">
        <property name="text">
         <string>Enable debug messages</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
